srv/fluffi/data/fluffiweb/app/templates/viewLogs.html

```diff
@@ -0,0 +1,78 @@
+<!--
+Copyright 2017-2019 Siemens AG
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Author(s): Junes Najah
+-->
+
+{% extends "base.html" %}
+{% block title %}FLUFFI - Logs{% endblock %}
+
+{% block scripts  %}
+<script src="{{ url_for('static', filename = 'javascript/pagination.js') }}" type="text/javascript"></script>
+{{super()}}
+{% endblock %}
+
+{% block content %}
+
+<div class="container">
+    <h1><u>Local Manager Logs:</u></h1>
+</div>
+
+<hr/>
+<div class="jumbotron">
+    <div class="container"></div>
+</div>
+
+<br>
+<br>
+
+<div class="container">       
+    {% for page in pages %}
+        {% if loop.index == 1 %}
+        <div id="page{{loop.index}}" class="activePage">
+        {% else %}
+        <div id="page{{loop.index}}" style="display: none;">
+        {% endif %}
+        <table class="table table-hover" style="font-size: 18px;">
+            <thead>
+                <tr>
+                    <th style="text-align: center;">ServiceDescriptorGUID</th>                
```

Comment:
Please add here fuzzjob and "host:port" as columns.

Reply:
Thanks!
I added Fuzzjob and host:port to the table on the feat_web_log_messages branch.